src-electron/util/types.js

```diff
@@ -78,6 +78,78 @@ async function typeSizeAttribute(db, zclPackageId, at, defaultValue = null) {
   }
 }
 
+/**
+ *
+ * @param {*} value
+ * @param {*} size
+ * @returns The negative hex value for a given positive hex value padded with
+ * the given size.
+ */
+function convertFloatToHex(value, size) {
+  let res = ''
+  const arrayBuffer = new ArrayBuffer(size)
+  const dataView = new DataView(arrayBuffer)
+  dataView.setFloat32(0, value, false)
```

Comment:
What if this is a 64-bit float?

Seems like this function should check for `size` being 4 or 8, throw if not one of those, then use the right accessors on the dataview.